server/src/frameworks/Projectless/ProjectlessProject.ts

```diff
@@ -31,7 +31,7 @@ export class ProjectlessProject extends Project {
   }
 
   public async fileBelongs(_file: string) {
-    return { belongs: true, isLocal: true };
```

Comment:
Can you walk me through this?

Is this change that we don't precache analysis on unclaimed solidity files under a project where the config has errored?